### PR TITLE
Add release note for v2.2

### PR DESCRIPTION
**Pull request by kvartet**

### CODE REVIEW

File: docs/en_US/Release.rst [hunk 1]
```diff
@@ -5,6 +5,62 @@
...
+* Support umount in remote training service for shared storage (#3456)
+* Support Windows as the remote training service in reuse mode (#3500)
+* Remove duplicated env folder in remote training service (#3472)
+* Add log information for GPU collector (#3506)
+
+WebUI
+"""""
+
+* Support launching TensorBoard on WebUI (#3454 #3361 #3531)
+* Minor improvements (#3457 #3461)
```
Comment (Lijiaoa) [inline docs/en_US/Release.rst:52]: I thought 
> Upgrade echarts-for-react to v5 #3457
Add wrap for dispatcher/nnimanager log monaco editor  #3461 

is better than `Minor imporvements`. It's too abstract.

Comment (kvartet) [reply docs/en_US/Release.rst]: updated~

File: docs/en_US/Release.rst [hunk 2]
```diff
@@ -5,6 +5,62 @@
...
+
+* Support speedup for mixed precision quantization model (Experimental) (#3488 #3512)
+* Support model export for quantization algorithm (#3458 #3473)
+* Support model export in model compression for TensorFlow (#3487)
+* Improve documentation (#3482)
+
+nnictl & nni.experiment
+"""""""""""""""""""""""
+
+* Add native support for v2 config (#3466 #3540)
```
Comment (QuanluZhang) [inline docs/en_US/Release.rst:37]: v2 config -> experiment config v2

File: docs/en_US/Release.rst [hunk 3]
```diff
@@ -5,6 +5,62 @@
...
+* Add resume and view mode in ``nni.experiment`` (#3490 #3524 #3545)
```
Comment (QuanluZhang) [inline docs/en_US/Release.rst:38]: in -> in python API

File: docs/en_US/Release.rst [hunk 4]
```diff
@@ -5,6 +5,62 @@
...
+
+Training Service
+""""""""""""""""
+
+* Support umount for shared storage in remote training service (#3456)
+* Support Windows as the remote training service in reuse mode (#3500)
+* Remove duplicated env folder in remote training service (#3472)
+* Add log information for GPU collector (#3506)
```
Comment (QuanluZhang) [inline docs/en_US/Release.rst:46]: GPU collector -> GPU metric collector

**Review by QuanluZhang — APPROVED** [review]

**Review by linbinskn — APPROVED** [review]

**Review by ultmaster — APPROVED** [review]

**Review by J-shang — APPROVED** [review]

File: docs/en_US/Release.rst (hunk 2 above)
Comment (kvartet) [reply docs/en_US/Release.rst]: updated all

**Review by Lijiaoa — APPROVED** [review]

**Pull request merged**